spaceship/docs/iTunesConnect.md

```diff
@@ -1,349 +1 @@
-submission.add_id_info_uses_idfa = false
-
-# Finalize app submission
-submission.complete!
-```
-
-For a full list of available options, check out [app_submission.rb](https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/tunes/app_submission.rb).
-
-## Testers
-
-There are 3 types of testers:
-
-- **External testers**: usually not part of your team. You can invite up to 10000 external testers. Before distributing a build to those testers you need to submit your app to beta review.
-- **Internal testers**: Employees that are registered in your App Store Connect team. They get access to all builds without having to wait for review.
-- **Sandbox testers**: Dummy accounts to test development-mode apps with in-app purchase or Apple Pay.
-
-```ruby
-# Find a tester based on the email address
-tester = Spaceship::TestFlight::Tester.find(app_id: "some_app_id", email: "felix@krausefx.com")
-
-# Creating new testers
-Spaceship::TestFlight::Tester.create_app_level_tester(
-      app_id: "io.myapp",
-       email: "github@krausefx.com",
-  first_name: "Felix",
-   last_name: "Krause"
-)
-
-```
-Right now, _spaceship_ can't modify or create internal testers.
-
-```ruby
-# Load all sandbox testers
-testers = Spaceship::Tunes::SandboxTester.all
-
-# Create a sandbox tester
-testers = Spaceship::Tunes::SandboxTester.create!(
-  email: 'sandbox@test.com', # required
-  password: 'Passwordtest1', # required. Must contain >=8 characters, >=1 uppercase, >=1 lowercase, >=1 numeric.
-  country: 'US', # optional, defaults to 'US'
-  first_name: 'Steve', # optional, defaults to 'Test'
-  last_name: 'Brule', # optional, defaults to 'Test'
-)
-
-# Delete sandbox testers by email
-Spaceship::Tunes::SandboxTester.delete!(['sandbox@test.com', 'sandbox2@test.com'])
-
-# Delete all sandbox testers
-Spaceship::Tunes::SandboxTester.delete_all!
-```
-
-## App ratings & reviews
-
-```ruby
-# Get the rating summary for an application
-ratings = app.ratings # => Spaceship::Tunes::AppRatings
-
-# Get the number of 5 star ratings
-five_star_count = ratings.five_star_rating_count
-
-# Find the average rating across all stores
-average_rating = ratings.average_rating
-
-# Find the average rating for a given store front
-average_rating = app.ratings(storefront: "US").average_rating
-
-# Get reviews for a given store front
-reviews = ratings.reviews("US") # => Array of hashes representing review data
-
-```
-
-### License
-
-> This project and all fastlane tools are in no way affiliated with Apple Inc. This project is open source under the MIT license, which means you have full access to the source code and can modify it to fit your own needs. All fastlane tools run on your own computer or server, so your credentials or other sensitive information will never leave your own computer. You are responsible for how you use fastlane tools.
+<h3 align="center">The <i>App Store Connect</i> docs were moved to <a href='AppStoreConnect.md'>AppStoreConnect.md</a></h3>
```

Comment:
💯

Reply:
Was a bit scared of this, but I tested on github.com on my fork before creating the PR 😃